types/database/schema/ChannelList.types.ts

```diff
@@ -2,6 +2,7 @@
 import {SQLiteDatabase} from 'react-native-sqlite-storage';
 
 import {ChatListDetail} from './ChatListDetail.types';
+import {SQLiteBoolean} from '../../../src/database/schema/UserSchema';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `SQLiteBoolean` seems to be incorrect. It's imported from `UserSchema`, but it should be imported from a more generic location if it's a type, not specific to the User schema.
```diff
- import {SQLiteBoolean} from '../../../src/database/schema/UserSchema';
+ import {SQLiteBoolean} from '../../../src/database/schema/TypeDefinitions';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->